package.json

```diff
@@ -18,6 +18,7 @@
         "prettier": "^1.16.4",
         "rimraf": "^2.6.3",
         "ts-jest": "^24.0.1",
+        "tslint": "^5.15.0",
```

Comment:
I'm always wondering if we should pin to a fixed version instead of using `^`. Thoughts?

Reply:
I'm open to discussing this further, but don't want to consider it part of this PR.

I think our use of a lock file covers the most important aspects of pinning to a fixed version, but agree it doesn't cover all aspects for why we might care. I'm not sure I'd want to add friction to considering upgrades without also adding something like greenkeeper to our process, though.